### PR TITLE
[DE] Add `Community/Video series/osu!mapping`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Community/Video_series/osu!mapping/de.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  - video
+  - videos
+  - YouTube
+---
+
+# osu!mapping
+
+*Nicht zu verwechseln mit [Beatmapping](/wiki/Beatmapping).*
+
+**osu!mapping** ist eine wöchentliche englische Videoreihe, die von ::{ flag=US }:: **[pishifat](https://osu.ppy.sh/users/3178418)** erstellt wurde und vom [Mapping-](/wiki/Beatmapping) und [Modding-](/wiki/Modding)Ökosystem handelt. Es wird auf dem [persönlichen YouTube-Kanal von pishifat](https://www.youtube.com/user/Pishifat) gehostet und die meisten Leute kennen es einfach als "pishifat videos" oder "pishi videos".
```
Comment (IceDynamix) [inline wiki/Community/Video_series/osu!mapping/de.md:16]:
```suggestion
**osu!mapping** ist eine wöchentliche englische Videoreihe, die von ::{ flag=US }:: **[pishifat](https://osu.ppy.sh/users/3178418)** erstellt wurde und vom [Mapping-](/wiki/Beatmapping) und [Modding-](/wiki/Modding)Ökosystem handelt. Es wird auf dem [persönlichen YouTube-Kanal von pishifat](https://www.youtube.com/user/Pishifat) gehostet und die meisten Leute kennen es einfach als "pishifat Videos" oder "pishi Videos".
```

probably better to capitalize

File: wiki/Community/Video_series/osu!mapping/de.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+Die Serie ist so bekannt geworden in der osu!-Community, dass sie eine allgemein empfohlene Informationsquelle für neue Mapper ist. Das [osu!-Team](/wiki/People/The_Team) verlieh pishifat das [Contributor-Abzeichen](/wiki/People/Community_Contributors) für eine "vorbildliche Leistung in der Beatmap-Erstellung und Weiterbildung" als Dank für seine Bemühungen, die er in das Projekt steckte.
```
Comment (IceDynamix) [inline wiki/Community/Video_series/osu!mapping/de.md:18]:
```suggestion
Die Serie ist in der osu!-Community so bekannt geworden, dass sie eine allgemein empfohlene Informationsquelle für neue Mapper ist. Das [osu!-Team](/wiki/People/The_Team) verlieh pishifat das [Contributor-Abzeichen](/wiki/People/Community_Contributors) für eine "vorbildliche Leistung in der Beatmap-Erstellung und Weiterbildung" als Dank für seine Bemühungen, die er in das Projekt steckte.
```

File: wiki/Community/Video_series/osu!mapping/de.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+## Links
+
+- **[YouTube-Kanal](https://www.youtube.com/user/Pishifat)**
+- [Patreon](https://www.patreon.com/pishifat) (aktuell geschlossen)
+- [Twitter](https://twitter.com/pishifat)
+
+## Geschichte
+
+osu!mapping begann als eine Reaktion auf einen [Reddit-Beitrag](https://www.reddit.com/r/osugame/comments/4upysl/request_for_mappers_of_this_sub/), mit der Aufforderung an Mapper, Videos mit Mapping-Kommentaren zu erstellen. Am 31.07.2016 lud pishifat das erste Video hoch: [osu!mapping: concepts](https://www.youtube.com/watch?v=oL3az8FhlB4). Im Video erklärt er, wie er den Kommentar erstellen wollte, wie der Reddit-Nutzer vorschlug, sich aber schließlich dagegen entschied, weil eine reine Erläuterung des Gedankengangs über Mapping nicht geeignet wäre, um neuen Mappern die große Vielfalt der Mapping-Konzepte näher zu bringen. Stattdessen entschied er sich, eine Videoserie zu starten, in der sich jede Folge auf ein bestimmtes Konzept konzentriert und dessen Verwendung und Bedeutung gesondert untersucht wird.
```
Comment (IceDynamix) [inline wiki/Community/Video_series/osu!mapping/de.md:30]:
```suggestion
osu!mapping begann als eine Reaktion auf einen [Reddit-Beitrag](https://www.reddit.com/r/osugame/comments/4upysl/request_for_mappers_of_this_sub/), mit der Aufforderung an Mapper, Videos mit Mapping-Kommentierung zu erstellen. Am 31.07.2016 lud pishifat das erste Video hoch: [osu!mapping: concepts](https://www.youtube.com/watch?v=oL3az8FhlB4). Im Video erklärt er, wie er die Kommentierung so erstellen wollte, wie der Reddit-Nutzer vorschlug, sich aber schließlich dagegen entschied, weil eine reine Erläuterung des Gedankengangs über Mapping nicht geeignet wäre, um neuen Mappern die große Vielfalt der Mapping-Konzepte näher zu bringen. Stattdessen entschied er sich dazu eine Videoserie zu starten, in der sich jede Folge auf ein bestimmtes Konzept konzentriert und dessen Verwendung und Bedeutung gesondert untersucht wird.
```

File: wiki/Community/Video_series/osu!mapping/de.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+Die Serie konzentrierte sich zunächst auf Lerninhalte, wurde aber nach einigen Monaten erweitert und umfasste nun auch Diskussionen über den  Zustand von Mapping und Modding, persönliche/meinungsstarke Kommentare und Interviews. Es wurden auch einige Forschungsprojekte in Form von osu!mapping-Videos vorgestellt, vor allem [eine Studie der Einflüsse von Mappern](https://www.youtube.com/watch?v=TigwDc5iOcw).
```
Comment (IceDynamix) [inline wiki/Community/Video_series/osu!mapping/de.md:32]:
```suggestion
Die Serie konzentrierte sich zunächst auf Lerninhalte, wurde aber nach einigen Monaten erweitert und umfasst nun auch Diskussionen über den Zustand von Mapping und Modding, persönliche/meinungsstarke Kommentare und Interviews. Es wurden auch einige Forschungsprojekte in Form von osu!mapping-Videos vorgestellt, vor allem [eine Studie der Einflüsse von Mappern](https://www.youtube.com/watch?v=TigwDc5iOcw).
```

File: wiki/Community/Video_series/osu!mapping/de.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+
+## Links
+
+- **[YouTube-Kanal](https://www.youtube.com/user/Pishifat)**
+- [Patreon](https://www.patreon.com/pishifat) (aktuell geschlossen)
+- [Twitter](https://twitter.com/pishifat)
+
+## Geschichte
+
+osu!mapping begann als eine Reaktion auf einen [Reddit-Beitrag](https://www.reddit.com/r/osugame/comments/4upysl/request_for_mappers_of_this_sub/), mit der Aufforderung an Mapper, Videos mit Mapping-Kommentierung zu erstellen. Am 31.07.2016 lud pishifat das erste Video hoch: [osu!mapping: concepts](https://www.youtube.com/watch?v=oL3az8FhlB4). Im Video erklärt er, wie er die Kommentierung so erstellen wollte, wie der Reddit-Nutzer vorschlug, sich aber schließlich dagegen entschied, weil eine reine Erläuterung des Gedankengangs über Mapping nicht geeignet wäre, um neuen Mappern die große Vielfalt der Mapping-Konzepte näher zu bringen. Stattdessen entschied er sich dazu eine Videoserie zu starten, in der sich jede Folge auf ein bestimmtes Konzept konzentriert und dessen Verwendung und Bedeutung gesondert untersucht wird.
```
Comment (The-Last-Cookie) [inline wiki/Community/Video_series/osu!mapping/de.md:30]:
```suggestion
osu!mapping begann als eine Reaktion auf einen [Reddit-Beitrag](https://www.reddit.com/r/osugame/comments/4upysl/request_for_mappers_of_this_sub/), mit der Aufforderung an Mapper, Videos mit Mapping-Kommentierung zu erstellen. Am 31.07.2016 lud pishifat das erste Video hoch: [osu!mapping: concepts](https://www.youtube.com/watch?v=oL3az8FhlB4). Im Video erklärt er, wie er die Kommentierung so erstellen wollte, wie der Reddit-Nutzer vorschlug, sich aber schließlich dagegen entschied, weil eine reine Erläuterung des Gedankengangs über Mapping nicht geeignet wäre, um neuen Mappern die große Vielfalt der Mapping-Konzepte näher zu bringen. Stattdessen entschied er sich dazu, eine Videoserie zu starten, in der sich jede Folge auf ein bestimmtes Konzept konzentriert und dessen Verwendung und Bedeutung gesondert untersucht wird.
```

**Pull request merged**